tests/host_tools/network.py

```diff
@@ -124,11 +125,7 @@ def run(self, cmd_string, timeout=None, *, check=False, debug=False):
 
         return self._exec(command, timeout, check=check)
 
-    def check_output(self, cmd_string, timeout=None, *, debug=False):
-        """Same as `run`, but raises an exception on non-zero return code of remote command"""
-        return self.run(cmd_string, timeout, check=True, debug=debug)
```

Comment:
We modeled `check_output/run` on `subprocess.check_output/run`. I think we should keep them as they are, as they are both useful. Most cases can be handled by check_output, but run is useful if the caller wants to handle the error.

Why not move more `run` calls to explicitly use `check_output` instead?

Reply:
Wouldn't it be easier if there was only 1 method to call? By default `run` will check output and if we need some exception, we will explicitly set `check=False`.
I did not touch `supbprocess.run` yet here, but was going to do same treatment there.

Reply:
I agree with Pablo. Ideally out `run`/`check_output` would behave exactly like `subprocess.run/check_output`.